### ISSUE TITLE
Passing Git Tag name as an environment variable

### ISSUE THREAD

**Issue by slyapustin** [digitalocean/app_action#106]
I'm deploying my app using such Github Action:

```yml
      - name: Deploy to Digital Ocean
        uses: digitalocean/app_action@v1.1.5
        with:
          app_name: my-cool-app
          token: ${{ secrets.DIGITALOCEAN_ACCESS_TOKEN }}
```
Is where a way I can pass GitHub tag name and git hash commit as an environment variable to the app?

Also, when it's deployed  can't see anything about what was deployed (git hash or version) in the DO App activity.

Is there an easy way to fix that?

**Comment by markusthoemmes** [comment]
We've just released v2.0.0 of the app which allows you to use environment-variable-subtitution for an app spec inside the repository. You can now pass whichever metadata you want to the app that way. Please check it out: https://github.com/digitalocean/app_action/discussions/122.

**Comment by slyapustin** [comment]
Awesome, thanks @markusthoemmes ! 
I'll take a look.
